src/lib/triggerAirflow.ts

```diff
@@ -0,0 +1,49 @@
+export interface TriggerDAGResponse {
+    dag_run_id: string;
+    conf: Record<string, unknown>;
+    state: string;
+    execution_date: string;
+}
+
+export async function triggerDAG(
+    directory: string,
+    inputfile: string
+): Promise<TriggerDAGResponse> {
+    const AIRFLOW_URL = "http://airflow.apps.l:8000";
+    const DAG_ID = "HIVE-run-pulse";
+    const username = "XXXXXXX";
+    const password = "XXXXXXX";
```

Comment:
We can use env vars here, similar to metacat
```suggestion
    AIRFLOW_URL
    AIRFLOW_DAG_ID
    AIRFLOW_USERNAME
    AIRFLOW_PASSWORD
```
These would be defined in the .env file

Reply:
Yeppp, nice, was thinking this! Will change this